### PR TITLE
Adding new api-version to Microsoft.Security Alerts resource

### PR DESCRIPTION
**Pull request by orparnes**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by hovsepm** [comment]
@orparnes please fix the build breaks

**Comment by hovsepm** [comment]
@orparnes please rework you PR in a way that the first commit will be a copy of existing swagger and then rest of the commits will be modifications. Otherwise it is impossible to understand what is the difference between API 2019-01-01 and 2015-06-01 (or whatever was the baseline).

**Comment by orparnes** [comment]
> @orparnes please rework you PR in a way that the first commit will be a copy of existing swagger and then rest of the commits will be modifications. Otherwise it is impossible to understand what is the difference between API 2019-01-01 and 2015-06-01 (or whatever was the baseline).

Just pushed another 2 commits:
First one is copy of "preview\2015-06-01-preview" into "stable\2019-01-01"
Second one is changes made to the new version and documentation fixes for the old one.

Those are the changes:

preview\2015-06-01-preview\security.json
- Under "AlertProperties", Add "isIncident" as it wasn't documented at all
- Under "AlertProperties", for "reportedSeverity", write all possible values (enum)

stable\2019-01-01
- Basically, creating a "stable" version for the "alerts" type, called it "2019-01-01".
- This is the first "stable" version of "Microsoft.Security", so added "stable" directory

stable\2019-01-01\security.json
- Remove all info that is not related to the "alerts" type.
- Under "AlertProperties", for "reportedSeverity", changed the possible values of the enum (this is the only actual change)

stable\2019-01-01\examples
those are the exact same examples of the "preview" version, except the "api-version" itself

Others stable versions for other "Microsoft.Security" types will be added after this first one.

**Comment by hovsepm** [comment]
@orparnes as I mentioned above please rework your PR. Plus you should not make changes in the stable PR in the new API version PR.

**Review by hovsepm — APPROVED** [review]

**Comment by ravbhatnagar** [comment]
There is another API-version for Security RP in review https://github.com/Azure/azure-rest-api-specs/pull/4978
@orparnes - Are you shipping 2 api versions and the changes in each are not part of the other API-version? Is this the intention? 
It may be better/quicker to get a review meeting scheduled (with armapireview) to go over the changes and the new APIs.

**Comment by ravbhatnagar** [comment]
Confirmed with service team that they are introducing a non-preview (GA) API version for the RP and want to support only alerts and location/alerts APIs in that. The only change from preview for these APIs is new values for AlertSeverity property.

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Comment by orparnes** [comment]
@hovsepm Can you please merge this PR?

**Comment by hovsepm** [comment]
@orparnes I've signed off already. @jianghaolu will make the final review and merge this PR.

**Review by chlahav — APPROVED** [review]

**Review by jianghaolu — APPROVED** [review]

**Comment by jianghaolu** [comment]
license/cla is not running. I'll try closing and reopening.
